castervoice/lib/ccr/core/nav.py

```diff
@@ -23,10 +23,9 @@
 class NavigationNon(MappingRule):
     mapping = {
         "<direction> <time_in_seconds>":
-            AsynchronousAction(
-                [L(S(["cancel"], Key("%(direction)s"), consume=False))],
-                repetitions=1000,
-                blocking=False),
+            AsynchronousAction([L(S(["cancel"], Key("%(direction)s"), consume=False))],
+                               repetitions=1000,
+                               blocking=False),
```

Comment:
Please revert these superfluous changes for now. We can fix style in a separate PR.